Assets/Scripts/Enemies/AIMovement.cs

```diff
@@ -12,16 +12,25 @@ public class AIMovement : MonoBehaviour
     public float currentDirection = 1f; 
     public float platformEdgeDistance = 2;
     public Enemy enemy;
+    public bool randomizeDirection = true;
+    public float randomizeDirectionTime = 1f;
     void Start()
 
     {
        parentCollider =  transform.parent.GetComponent<Collider2D>();  
        m_Rigidbody = GetComponent<Rigidbody2D>();
        enemy = GetComponentInParent<Enemy>();
 
-  
+        if( randomizeDirection){
+        InvokeRepeating("randomlyChangeDirection", 0, randomizeDirectionTime);
+        }
```

Comment:
Minor suggestion and purely preferential coding style: we can use a Coroutine for this:

```csharp
IEnumerator RandomlyChangeDirection() {
  if(Random.Range(0, 100) > 95){
    currentDirection *= -1;
  }
  yield return new WaitForSeconds(randomizeDirectionTime);
}
```

Then, the above would become:

```suggestion
        if( randomizeDirection){
          StartCoroutine(RandomlyChangeDirection());
        }
```

Reply:
I actually like this suggestion, we can then change it during gameplay. 
Added.